opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -152,3 +152,21 @@ def _python_path_without_directory(python_path, directory, path_separator):
         "",
         python_path,
     )
+
+
+def is_instrumentation_enabled() -> bool:
+    if (
+        context.get_value("suppress_instrumentation")
+        or 
+        context.get_value(context._SUPPRESS_INSTRUMENTATION_KEY)  # type: ignore
+    ):
+        return False
+    return True
+
+
+def is_http_instrumentation_enabled() -> bool:
+    return (
+        is_http_instrumentation_enabled()
```

Comment:
did you mean `is_instrumentation_enabled`?

Reply:
Probably! Still need to run linting etc on this